src/main/java/com/lowcd/TCS/controller/UserController.java

```diff
@@ -0,0 +1,50 @@
+package com.lowcd.TCS.controller;
+
+import com.lowcd.TCS.model.LoginBO;
+import com.lowcd.TCS.entity.Role;
+import com.lowcd.TCS.entity.User;
+import com.lowcd.TCS.model.UserBO;
+import com.lowcd.TCS.repository.RoleRepository;
+import com.lowcd.TCS.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@CrossOrigin
+@RequestMapping("api/v1/user")
+
+
+
+public class UserController {
+
+    @Autowired
+    private UserService userservice;
+    @Autowired
+    private RoleRepository roleRepository;
+
+  @PostMapping(path="/save")
+  public String saveUser(@RequestBody UserBO userBO)
+  {
+
```

Comment:
add logic to check if a user already exists with the given mail, if already exists then send the respective response